UmpleCollabServer/src/server.ts

```diff
@@ -153,14 +145,10 @@ io.on('connection', (socket: Socket) =>{
   // when the client requests for them to be pulled
   // fileKey parameter should come from the client - umpdir_filename
   socket.on('pullUpdates', (fileKey: string, version: number) => {
-    // DEBUG
-    // console.log("inside pullUpdates with filekey: ", fileKey)
 
     if(serverDebugFlag){
```

Comment:
We can name this variable **collabServerDebugFlag** - just to signify it is part of server side - specifically for collab server

Reply:
Sure, I will change it to that.